flocker/node/functional/sendbytes-docker/Dockerfile.in

```diff
@@ -1,4 +1,6 @@
 FROM busybox
 MAINTAINER ClusterHQ <support@clusterhq.com>
 ADD . /
+# If installed via wheel, the installed file will not have +x set.
+RUN ["chmod", "+x", "/run.sh"]
```

Comment:
New versions of pip install by building a wheel. Since zip files (which is what underlies wheels) don't record `+x`, when the package gets installed, these scripts get installed without `+x`. So we need to set it here.